### PR TITLE
feat: update android guide and example

### PR DESCRIPTION
**Pull request by ferdicus**
## Description
Updates android install guide and `/example` projects `build.gradle`

### Additional Info
People always come up to me and ask "ferdicus, how can I install custom version of mapbox" or they yell "yo, how to mapLibre" across the street.  
Thought it would be a good idea to clean up the android install guide and add example install configs to the ... `/example`... app. 

And here we are.

## Checklist

- [x] I have tested this on a device/simulator for each compatible OS
  - tested with default project dependencies
  - tested with up to date mapbox dependencies as commented out within `/example` > `build.gradle`
  - tested with mapLibre dependencies as included within `/example` > `build.gradle`
- [x] I mentioned this change in `CHANGELOG.md`

### not applicable tests

- [ ] I formatted JS and TS files with running `yarn lint:fix` in the root folder 
- [ ] I updated the documentation with running `yarn generate` in the root folder 
- [ ] I updated the typings files (`index.d.ts`)
- [ ] I added/ updated a sample (`/example`)

### CODE REVIEW

**Review by lisabaut — APPROVED** [review]
LGTM

File: example/android/build.gradle [hunk 1]
```diff
@@ -10,15 +10,32 @@ buildscript {
 
         useMapLibre = false
 
+        // Mapbox deps
```
Comment (lisabaut) [inline example/android/build.gradle:13]: But this has to be commented out, right?

Comment (lisabaut) [reply example/android/build.gradle:13]: Maybe add a comment why?

Comment (ferdicus) [reply example/android/build.gradle:13]: oh, even better, I can add the same logic like we have for `useMapLibre` 

Thanks 👍🏿

**Pull request merged**